### PR TITLE
Query CSV data with Flux

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/flux/v0.x/query-data/csv.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+{{% /note %}}
+
+## Results structure
+The structure of results returned by `csv.from()` depends on the
+[parsing mode](#csv-parsing-modes) used.
+
+- **annotations**: `csv.from()` returns a stream of tables grouped by columns
+  defined as `true` in the `#group` annotation row.
+- **raw**: `csv.from()`returns a stream of tables with no grouping (all rows are in a single table).
+  All data is formatted as strings.
```
Comment (kelseiv) [inline content/flux/v0.x/query-data/csv.md:70]: @sanderson thoughts on pulling overview content into separate doc, so folks can jump into procedures:
- Query an annotated CSV string
- Query a raw CSV string
- Query CSV data from a file
- Query CSV data from a URL

File: content/flux/v0.x/query-data/csv.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+## Examples
```
Comment (kelseiv) [inline content/flux/v0.x/query-data/csv.md:72]: Maybe nice to say on top-level pages:
If you're just getting started, see how to run this Flux query in xyz....and link back to that doc

File: content/flux/v0.x/query-data/csv.md [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+,,2,air-sensors,temperature,TLM0100,1627049700000000000,71.87
+,,3,air-sensors,temperature,TLM0200,1627049400000000000,74.10
+,,3,air-sensors,temperature,TLM0200,1627049700000000000,74.17
+"
+
+csv.from(csv: csvData)
+```
+
+#### Results
+| dataset     | metric   | sensorID | timestamp           | value |
```
Comment (kelseiv) [inline content/flux/v0.x/query-data/csv.md:107]: Shall we add table 0, 1, 2, 3 in results?

File: content/flux/v0.x/query-data/csv.md [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+| air-sensors | temperature | TLM0200  | 1627049400000000000 | 74.10 |
+| air-sensors | temperature | TLM0200  | 1627049700000000000 | 74.17 |
+
+---
+
+### Query CSV data from a file
+Use `csv.from()` and the `file` parameter to query CSV data from a file.
+
+{{% note %}}
+To query CSV data from a file, Flux must have access to the filesystem.
```
Comment (kelseiv) [inline content/flux/v0.x/query-data/csv.md:174]: how can folks test access?

File: content/flux/v0.x/query-data/csv.md [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+
+### Query CSV data from a URL
+Import the [`experimental/csv` package](/flux/v0.x/stdlib/experimental/csv/)
+and use the [experimental `csv.from()` function](/flux/v0.x/stdlib/experimental/csv/from/)
+to query data from a URL.
+Use the `url` parameter to specify the URL to query.
+
+{{% note %}}
+The experimental `csv.from()` function does not support multiple parsing modes
+and only works with annotated CSV.
```
Comment (kelseiv) [inline content/flux/v0.x/query-data/csv.md:233]: shall we link to annotated CSV?

**Review by kelseiv — APPROVED** [review]

File: content/flux/v0.x/query-data/csv.md (hunk 4 above)
Comment (sanderson) [reply content/flux/v0.x/query-data/csv.md:174]: Probably just trying to execute the query. I'll add the error output here so if people search for it, it'll land them here.

File: content/flux/v0.x/query-data/csv.md (hunk 5 above)
Comment (sanderson) [reply content/flux/v0.x/query-data/csv.md:233]: 👍🏻

File: content/flux/v0.x/query-data/csv.md [hunk 6]
```diff
@@ -0,0 +1,268 @@
...
+#### Flux must have access to the file system
+To query CSV data from a file, Flux must have access to the filesystem.
+If Flux does not have access to the file system, the query will return an error
+simlir to:
+
+```
+failed to read file: filesystem service is uninitialized
+```
+If using InfluxDB Cloud or InfluxDB OSS, the Flux process **does not** have 
+access to the filesystem.
```
Comment (sanderson) [inline content/flux/v0.x/query-data/csv.md:187]:
~~~suggestion
If Flux does not have access to the file system, the query will return an error
similar to:

```
failed to read file: filesystem service is uninitialized
```

If using InfluxDB Cloud or InfluxDB OSS, the Flux process **does not** have 
access to the filesystem.
~~~

**Pull request merged**